### ISSUE TITLE
adding this extension to extensions.gnome.org

### ISSUE THREAD

**Issue by ppr1** [simon04/gnome-shell-extension-weather#89]
Hi,

Should be a good idea (and not very time consuming) to add this extension to the website https://extensions.gnome.org. This website aims to offer all extension for gnome.

On some system which doesn't have easy way to install this extension (debian for example), it's would avoid to compile this program.

And it would allow to install/uninstall this extension in one clic (for testing).

**Comment by yellowhat** [comment]
I agree with you. Would be fantastic have this extension on extensions.gnome.org

**Comment by Kysic** [comment]
Hi,
This issue is already referenced in : #50
I would be pleased to see it on extensions.gnome.org too.
